### DISCUSSION TITLE
Line breaks are not inserted

### DISCUSSION THREAD

**Discussion by kertAW** [remarkjs/remark#677] [Q&A] [answered]
I have a line:

![image](https://user-images.githubusercontent.com/64747555/113158548-9038b900-9244-11eb-9281-ae7fb614af77.png)

This line transform into:

![image](https://user-images.githubusercontent.com/64747555/113158566-94fd6d00-9244-11eb-9e60-125a274d6feb.png)

Code:

```
<div className="text">
  <ReactMarkdown plugins={[gfm]} children={line} />
</div>
```

Question: where did the line breaks go? How to fix?

**Comment by wooorm** [comment, marked as answer]
This is how markdown (and HTML, React) works. You don’t need line breaks

**Reply by kertAW** [reply]
@wooorm, Didn't figure it out a bit, thanks
